readme.md

```diff
@@ -80,7 +80,7 @@ The Chrome version also works in Opera (using [this](https://addons.opera.com/en
 			Linkifies issue/PR references and URLs in code
 		</th>
 		<th width="50%">
-			<a title="revert-file"></a> Adds button to revert all the changes to a file in a PR.
+			<a title="restore-file"></a> Adds button to restore all the changes to a file in a PR.
```

Comment:
Changes aren’t restored; the file is restored. 

The previous description was clear enough, we can restore it. 😃